mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Xcribe.MixProject do
   use Mix.Project
 
-  @version "0.0.2"
+  @version "0.1.0"
```

Comment:
@brosquinha Could you please also update the readme to be in sync with the new version?

Reply:
Done! :+1: